src/test/java/com/google/step/servlets/UserServletTest.java

```diff
@@ -114,33 +123,33 @@ public void testDoGet_noId() throws Exception {
   public void testDoPost_userNotLoggedIn() throws Exception {
     HttpServletRequest request = mock(HttpServletRequest.class);
     HttpServletResponse response = mock(HttpServletResponse.class);
-    when(userService.isUserLoggedIn()).thenReturn(false);
+    when(mockUserService.isUserLoggedIn()).thenReturn(false);
 
-    servlet.doPost(request, response);
+    mockUserServlet.doPost(request, response);
     verify(response).setStatus(HttpServletResponse.SC_UNAUTHORIZED);
   }
 
   @Test
   public void testDoPost_success() throws Exception {
```

Comment:
Seems like there's no test to ensure the new tags passed in the `POST` request are updated?